interpreter/cling/lib/Interpreter/Interpreter.cpp

```diff
@@ -1031,6 +1031,31 @@ namespace cling {
     return getDynamicLibraryManager()->lookupLibrary(canonicalFile);
   }
 
+  void Interpreter::replaceEnvVars(std::string &Path){
+
+    std::size_t fpos = Path.find("$");
+
+    while(fpos != std::string::npos) {
+      std::size_t spos = Path.find("/", fpos + 1);
+      std::size_t length;
```

Comment:
Can you initialize this to `length = Path.length()`? (We usually use `length()` not `size()`.)